build.sh

```diff
@@ -102,11 +102,11 @@ function all_frameworks_test() {
 function strip_framework() {
   local FRAMEWORK_PATH="$BUILD_DIRECTORY/Build/Products/Debug/$1"
   echo "Stripping Framework $FRAMEWORK_PATH"
-  rm -r "$FRAMEWORK_PATH"
+  rm -rf "$FRAMEWORK_PATH"
```

Comment:
I doubt that you would want to use `-f`! Could lead to some really nasty behaviour.